src/inputs/Button/Button.module.css

```diff
@@ -1,58 +1,86 @@
-.primary {
-  color: #000000;
-  border: 1px solid #000000;
+.buttons {
+	min-width: 103px;
+	font-weight: 500;
+	padding: 10px 24px;
+	font-size: 14px;
+	line-height: 20px;
 }
 
-.secondary {
-  border: 1px solid #90a8ed;
-  color: #90a8ed;
+button:hover .iconSettings svg path {
+	fill: #ffffff;
 }
-
-.template {
-  font-style: normal;
-  font-weight: 500;
-  background: #ffffff;
-  font-size: 14px;
-  min-width: 103px;
-  padding: 10px 24px;
-  line-height: 20px;
+.button__container {
+	display: flex;
 }
-
-.disabledSecondary {
-  border: 1px solid rgba(28, 27, 31, 0.12);
-  opacity: 0.38;
+.primary {
+	color: black;
+	background-color: #ffffff;
+	border: 1px solid #000000;
 }
 
 .primary:hover {
-  background: #b23386;
-  border: 1px solid #000000;
-  color: #ffffff;
-  box-shadow: 4px 4px 0px #000000;
+	background: #b23386;
```

Comment:
**nitpick:** We have to start thinking about CSS variables to stop duplicating these values